diff --git a/src/luaconf.h b/src/luaconf.h
index d283233..f3c66bd 100644
--- a/src/luaconf.h
+++ b/src/luaconf.h
@@ -27,13 +27,13 @@
 ** Please read ../doc/install.html#distro and pass PREFIX=/usr instead.
 */
 #define LUA_ROOT	"/usr/local/"
-#define LUA_LDIR	LUA_ROOT "share/lua/5.1/"
-#define LUA_CDIR	LUA_ROOT "lib/lua/5.1/"
+#define LUA_LDIR	LUA_ROOT "share/lua/"
+#define LUA_CDIR	LUA_ROOT "lib/lua/"
 #ifdef LUA_XROOT
 #define LUA_JDIR	LUA_XROOT "share/luajit-2.1.0-alpha/"
 #define LUA_XPATH \
-  ";" LUA_XROOT "share/lua/5.1/?.lua;" LUA_XROOT "share/lua/5.1/?/init.lua"
-#define LUA_XCPATH	LUA_XROOT "lib/lua/5.1/?.so;"
+  ";" LUA_XROOT "share/lua/?.lua;" LUA_XROOT "share/lua/?/init.lua"
+#define LUA_XCPATH	LUA_XROOT "lib/lua/?.so;"
 #else
 #define LUA_JDIR	LUA_ROOT "share/luajit-2.1.0-alpha/"
 #define LUA_XPATH
